### PR TITLE
Updated P[T>t] description, managed to put system MM1 Balking after S…

### PR DESCRIPTION
**Pull request by Szabin96**
…ystem MM1, added system utilization property. (QSA: #225)

### CODE REVIEW

File: qsa-application-resources/src/main/resources/features/features.json [hunk 1]
```diff
@@ -59,6 +59,12 @@
 		"description": "Expected customer service time, E[S<sub>3</sub>]",
 		"orderNumber": "2.03"
 	},
+	{
+		"id": "US",
+		"name": "U<sub>s</sub>]",
```
Comment (ZoltanSzilagyiCse) [inline qsa-application-resources/src/main/resources/features/features.json]: Missing a start `[`

File: qsa-application-resources/src/main/resources/features/features_hu.json [hunk 2]
```diff
@@ -49,6 +49,11 @@
 		"name": "<span class=\"overline\">S</span><sub>3</sub>",
 		"description": "Az igény átlagos kiszolgálási ideje 3, E[S<sub>3</sub>]"
 	},
+	{
+		"id": "US",
+		"name": "U<sub>s</sub>]",
```
Comment (ZoltanSzilagyiCse) [inline qsa-application-resources/src/main/resources/features/features_hu.json]: Here too.

File: qsa-application-resources/src/main/resources/features/features.json (hunk 1 above)
Comment (Szabin96) [reply qsa-application-resources/src/main/resources/features/features.json]: Its not missing. In the end I leaved it there by mistake.

File: qsa-application-resources/src/main/resources/features/features_hu.json (hunk 2 above)
Comment (Szabin96) [reply qsa-application-resources/src/main/resources/features/features_hu.json]: Same here.

**Pull request merged**